### PR TITLE
added recipe for Abydos

### PR DESCRIPTION
**Pull request by chrislit**

### CODE REVIEW

File: recipes/abydos/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  imports:
+    - abydos
+
+about:
+  home: https://github.com/chrislit/abydos
+  license: GNU General Public v3 or later (GPLv3+)
+  license_family: GPL3
+  license_file:
```
Comment (marcelotrevisani) [inline recipes/abydos/meta.yaml]: Please, add the ``license_file``

Comment (marcelotrevisani) [reply recipes/abydos/meta.yaml]:
```
license_file: LICENSE
```

File: recipes/abydos/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+
+requirements:
+  host:
```
[the remark this answers is not in the document]
Comment (chrislit) [reply recipes/abydos/meta.yaml]: Done -- Thanks for the tip

**Comment by marcelotrevisani** [comment]
I think you can merge this as it is ``noarch`` and ``appveyor`` is quite busy these days.

Any problem please ping us.

Thanks to add this in ``conda-forge``! :tada:

**Pull request merged**